program/src/processor/validator_stake_tokens.rs

```diff
@@ -114,25 +110,32 @@ pub fn process_stake_tokens<'a>(
     let decimals = mint.base.decimals;
 
     // Update the config and stake account data.
-    //
-    // Note: validate the amount against the total SOL staked on the validator.
 
     require!(amount > 0, StakeError::InvalidAmount);
 
-    require!(amount > 0, StakeError::InvalidAmount);
```

Comment:
What happened here in the old code? :sweat_smile:

Reply:
I was equally surprised. 😅 I think this is a result of having the same commit in separate PRs due to "squash and merge" that we discussed the other day.